index.html

```diff
@@ -177,7 +177,47 @@ <h2>Multi-Post Stories</h2>
                 <button class="btn-1">See Project</button>
               </div>
             </div>
+          </div>
+        </div>
+          <div class="sub-grid-4">
+            <div class="sub-flex-2">
+              <div class="box">
+                <input type="text">
+              </div>
+              <div class="story-title">
+                <div>
+                  <h2>Multi-Post Stories</h2>
+                  <p>A daily selection of privately<br> personalized read; no accounts<br> or sign-ups required, has been
+                  the<br> industry's standard dummy text<br> ever since the 1500s, when an<br> unknown printer took a
+                  took a standard<br> dummy text</p>
+                </div>
+                <div>
+                  <ul>
+                    <li>css</li>
+                    <li>html</li>
+                    <li>bootstrap</li>
+                    <li>Ruby</li>
+                  </ul>
+                </div>
+                <div>
+                  <button class="btn-1">See Project</button>
+                </div>
+              </div>
+            </div>
+          </div>
       </div>
+            <div class="sub-grid-5">
+              <h2 class="text-align">CONTACT US</h2>
+              <p class="text-align">If you have an application you are<br> interested in developing, a feature that you<br> need
+                built or a project that needs coding.<br> I’d love to help with it</p>
+                <form class="text-align" action="/Ademola-Portfolio" method="post">
```

Comment:
- [ ] Please make sure that your form form uses the action and method attribute to integrate with [formspree service(https://formspree.io/html). Please open the link and check the documentation on how to do that. The action value should be similar to `https://formspree.io/f/{form_id}`, eg:    `action="https://formspree.io/f/xyyodjya"`